extensions/amp-analytics/0.1/visibility-model.js

```diff
@@ -126,8 +133,12 @@ export class VisibilityModel {
     /** @private {time} milliseconds since epoch */
     this.lastVisibleUpdateTime_ = 0;
 
-    /** @private {boolean} */
-    this.waitToReset_ = false;
+    /**
+     * Want to fire when visibility becomes zero, *after* having been
+     * non-zero. So, set up the state as though it needs to wait to fire.
+     * @private {boolean} */
+    this.waitToReset_ = this.spec_.visiblePercentageMin == 0 &&
```

Comment:
if user explicitly put `visiblePercentageMax=0`, i think we should fire if it's not visible on page load.

Reply:
Will do (including changing test to expect this)